stdlib/public/Cxx/CxxDictionary.swift

```diff
@@ -72,4 +77,22 @@ extension CxxDictionary {
       }
     }
   }
+  
+public func filter(_ isIncluded: (_ key: Key, _ value: Value) throws -> Bool) rethrows -> Self {
```

Comment:
Please adjust indentation on this line

Reply:
Looks like the formatting isn't quite correct, let's have a two-space indentation and a blank line before this method